src/pkg/selectors/groups.go

```diff
@@ -770,6 +770,19 @@ func (s GroupsScope) matchesInfo(dii details.ItemInfo) bool {
 		return false
 	}
 
+	acceptableItemType := -1
+
+	switch infoCat {
```

Comment:
This seems a lot like [`leafCat`](https://github.com/alcionai/corso/blob/ce422f0b1d1d92756731b396290198a100fc4b25/src/pkg/selectors/groups.go#L556). Maybe we can reuse that?

Reply:
Hey!  I completely forgot about that 😅.  Thanks for the reminder, this is exactly what it's here for.